example/push_manifest.json

```diff
@@ -1,10 +0,0 @@
-{
```

Comment:
I think I wanted to ignore it from the main folder, but not here. Can you make that change? The idea being, the example py server is ready to run out of the box.